[2024-10-01T07:36:11.671+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T07:36:11.684+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_2.check_if_table_exists manual__2024-10-01T07:36:07.934698+00:00 [queued]>
[2024-10-01T07:36:11.689+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_2.check_if_table_exists manual__2024-10-01T07:36:07.934698+00:00 [queued]>
[2024-10-01T07:36:11.689+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T07:36:11.697+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_if_table_exists> on 2024-10-01 07:36:07.934698+00:00
[2024-10-01T07:36:11.700+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=86) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-01T07:36:11.701+0000] {standard_task_runner.py:72} INFO - Started process 88 to run task
[2024-10-01T07:36:11.702+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_2', 'check_if_table_exists', 'manual__2024-10-01T07:36:07.934698+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/dag_id_2.py', '--cfg-path', '/tmp/tmpgp9jhc8t']
[2024-10-01T07:36:11.703+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask check_if_table_exists
[2024-10-01T07:36:11.743+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_id_2.check_if_table_exists manual__2024-10-01T07:36:07.934698+00:00 [running]> on host f5d39094eeb7
[2024-10-01T07:36:11.795+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_2' AIRFLOW_CTX_TASK_ID='check_if_table_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T07:36:07.934698+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T07:36:07.934698+00:00'
[2024-10-01T07:36:11.796+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-01T07:36:11.796+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-01T07:36:11.796+0000] {logging_mixin.py:190} INFO - Current task name:check_if_table_exists state:running start_date:2024-10-01 07:36:11.684724+00:00
[2024-10-01T07:36:11.797+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_2 and current dag run status:running
[2024-10-01T07:36:11.797+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T07:36:11.797+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-01T07:36:11.798+0000] {python.py:240} INFO - Done. Returned value was: create_table
[2024-10-01T07:36:11.798+0000] {branch.py:38} INFO - Branch into create_table
[2024-10-01T07:36:11.798+0000] {skipmixin.py:230} INFO - Following branch ('create_table',)
[2024-10-01T07:36:11.810+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2024-10-01T07:36:11.831+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T07:36:11.832+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_id_2, task_id=check_if_table_exists, run_id=manual__2024-10-01T07:36:07.934698+00:00, execution_date=20241001T073607, start_date=20241001T073611, end_date=20241001T073611
[2024-10-01T07:36:11.838+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-01T07:36:11.838+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-01T07:36:11.838+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_2 queued_at:2024-10-01 07:36:07.949285+00:00
[2024-10-01T07:36:11.839+0000] {logging_mixin.py:190} INFO - Task hostname:f5d39094eeb7 operator:BranchPythonOperator
[2024-10-01T07:36:11.844+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-01T07:36:11.864+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-01T07:36:11.865+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
